docs/release/release_0_3_5.md

```diff
@@ -0,0 +1,59 @@
+# napari 0.3.5
+
+We're happy to announce the release of napari 0.3.5!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
```

Comment:
I would add a highlights section here, something like:

```
## Highlights

This release contains a number of bug fixes on various platforms. For those interested
in napari performance, we have added a new performance monitoring mode, that can
be activated by <method + link to relevant docs>. We have also added a page on
napari's rendering pipeline [here](<link to new rendering page>)
```

Currently I can't find either of those doc links, though. I'll raise an issue.

Reply:
See #1365

Reply:
I have added this but only referenced the PRs until #1365 is fixed